types/screeps/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
+        "file-name-casing": false,
```

Comment:
Why'd this change?

Reply:
Was leftover of a direct copy from our [local environment](https://github.com/screepers/typed-screeps/blob/master/dist/tslint.json). I'll remove it.